### ISSUE TITLE
Add sort direction icon for variant options table

### ISSUE THREAD

**Issue by bjarnef** [TeaCommerce/Tea-Commerce-for-Umbraco#46]
I think it would be great to add a sort direction icon for ascending/descending like in listview instead of ```icon-navigation-road``` - so it will be more clear than sorting in happening, e.g. if sorting on a column with same values.

**Comment by rasmusfjord** [comment]
I would agree, an option to drag in the variant table instead of just an asc/desc would be nice aswell!
